lib/cmake/cccl/cccl-config.cmake

```diff
@@ -36,6 +40,12 @@ foreach(component IN LISTS components)
         "${cccl_cmake_dir}/../../../libcudacxx/lib/cmake/" # Source layout (GitHub)
         "${cccl_cmake_dir}/.."                             # Install layout
     )
+    # Can't alias other alias targets, so use the uglified target name instead
+    # of libcudacxx::libcudacxx:
+    if (TARGET _libcudacxx_libcudacxx)
+      add_library(CCCL::libcudacxx ALIAS _libcudacxx_libcudacxx)
```

Comment:
Need to guard this on the CCCL::libcudacxx target already existing

Reply:
Good catch -- fixing.